parquet/src/main/java/org/apache/iceberg/parquet/Parquet.java

```diff
@@ -136,6 +138,15 @@ private Parquet() {}
           "parquet.read.support.class",
           "parquet.crypto.factory.class");
 
+  private static final int MAX_RECORDS_PER_BATCH_DEFAULT = 10000;
+
+  public static final String WRITER_VERSION_KEY = "parquet.writer.version";
+
+  /**
+   * @deprecated Since 1.10.0, will be removed in 1.11.0. Use {@link
+   *     FormatModelRegistry#writeBuilder(FileFormat, String, EncryptedOutputFile)} instead.
+   */
+  @Deprecated
```

Comment:
This should not be deprecated. I think that we want Parquet to continue to expose this without going through the registry. This is a valid thing for Parquet to expose.

Reply:
My rationale is that this module contains code for working with Parquet using Iceberg constructs like schema and table properties. We want data and delete files to be written through the new interfaces, but this is just creating a Parquet file writer for some purpose. This seems like a core part of what the module does so I would not force people to register and use an object model to create a Parquet file.

Reply:
Let's finalize the API, then change this PR accordingly